core/src/main/scala/cats/data/OptionT.scala

```diff
@@ -63,6 +86,13 @@ final case class OptionT[F[_], A](value: F[Option[A]]) {
   def subflatMap[B](f: A => Option[B])(implicit F: Functor[F]): OptionT[F, B] =
     transform(_.flatMap(f))
 
+  /**
+   * Perform an effect if the value inside the is a [[None]], leaving the result untouched. Equivalent to [[orElseF]]
+   * with an effect returning [[None]] as argument.
+   */
+  def emptyflatTap[B](f: => F[B])(implicit F: Monad[F]): OptionT[F, A] =
```

Comment:
We could require `F[Unit]` here, but that would be inconsistent with `FlatMap.flatTap` which has the type parameter as well.

The name with lowercase `flat` follows the example of `subflatMap` and `semiflatMap`.

Reply:
Yes, I'd keep the type parameter for consistency. Having similar things grouped by prefix is nice for discovery through autocompletion or browsing the scaladoc, so maybe `flatTapNone`?

Reply:
I agree with the autocompletion argument, so renamed to `flatTapNone`, thanks for the name :)